HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -3639,14 +3640,29 @@ def self.calc_sequential_load_fraction(load_fraction, remaining_fraction)
     return sequential_load_frac
   end
 
-  def self.get_sequential_load_schedule(model, value)
-    s = OpenStudio::Model::ScheduleConstant.new(model)
-    s.setName('Sequential Fraction Schedule')
-    if value > 1
-      s.setValue(1.0)
+  def self.get_sequential_load_schedule(model, value, multipliers)
+    values = []
+    multipliers.each do |mult|
+      values.push(value * mult)
+    end
+
+    values << value if values.empty?
+
+    if values.uniq.length == 1
+      s = OpenStudio::Model::ScheduleConstant.new(model)
+      s.setName('Sequential Fraction Schedule')
+      if values[0] > 1
+        s.setValue(1.0)
+      else
+        s.setValue(values[0].round(5))
+      end
```

Comment:
This logic needs to apply to both if/else branches, not just here. It was added to prevent values like 1.0000001 from being written to EnergyPlus, which results in simulation errors.